lib/property_table/supervisor.ex

```diff
@@ -54,6 +25,6 @@ defmodule PropertyTable.Supervisor do
   @doc false
   @spec registry_name(PropertyTable.table_id()) :: Registry.registry()
   def registry_name(name) do
-    Module.concat(PropertyTable.Registry, name)
+    Module.concat(name, Registry)
```

Comment:
```suggestion
    Module.concat(name, Subscriptions)
```
Just an idea - maybe call this for what the registry use is?

Reply:
Yeah, that makes a lot of sense.